addons/minedetector/XEH_clientInit.sqf

```diff
@@ -0,0 +1,40 @@
+#include "script_component.hpp"
+
+
+[QGVAR(detectorEnabled), {
```

Comment:
Those 2 events are only called locally, why are they events in the first place? Either use direction function call or make them public?